gap/PolygonalComplexes/animating.gi

```diff
@@ -797,170 +806,302 @@ InstallMethod( GetCircleColours,
-                        AppendTo(output, "\t\tface", faces[i], ".faces.push(new THREE.Face3(0, 1, 2 ,undefined, undefined, 0));\n");
-                        AppendTo(output, "\t\tvar face", faces[i], "_obj = new THREE.Face3(0,1,2,undefined, undefined, 0);\n");
-                        AppendTo(output, "\t\tobj.add( new THREE.Mesh(face", faces[i], ", face", faces[i], "_material) );\n");
-                    fi;
-                od;
-
-                AppendTo(output, "\n\n");
-                for face in faces do
-                    if IsInnerCircleActive(surface, face, printRecord) then
-                        parametersOfCircle := printRecord.innerCircles[face];
-                        colour := GetCircleColour(surface, face, printRecord);
-                        if not StartsWith(colour, "0x") then
-                            colour := Concatenation("\"", colour, "\"");
-                        fi;
-                        AppendTo(output, "\t\tvar circle = Circle(", parametersOfCircle[2], ", ", parametersOfCircle[1][1], ", ",
-                            parametersOfCircle[1][2], ", ", parametersOfCircle[1][3], ", ", parametersOfCircle[3][1], ", ",
-                            parametersOfCircle[3][2], ", ", parametersOfCircle[3][3], ", ", colour, ");\n");
-                        AppendTo(output, "\t\tobj.add(circle);\n");
-                    fi;
-                od;
-                AppendTo(output, "\n\n");
-
-                for face in faces do
-                    if IsNormalOfInnerCircleActive(surface, face, printRecord) then
-                        parametersOfCircle := printRecord.innerCircles[face];
-                        colour := GetCircleColour(surface, face, printRecord);
-                        if not StartsWith(colour, "0x") then
-                            colour := Concatenation("\"", colour, "\"");
-                        fi;
-                        AppendTo(output, "\t\tvar normalVector = Edge(", parametersOfCircle[5], ", 0.002, ", parametersOfCircle[1][1], ", ",
-                            parametersOfCircle[1][2], ", ", parametersOfCircle[1][3], ", ", parametersOfCircle[4][1], ", ",
-                            parametersOfCircle[4][2], ", ", parametersOfCircle[4][3], ", ", colour, ");\n");
-                        AppendTo(output, "\t\tobj.add(normalVector);\n");
-                    fi;
-                od;
-                AppendTo(output, "\n\n");
+        edgeThickness := printRecord.edgeThickness*100;
+        AppendTo(output, """
+            const edgeMaterial""",i,""" = new THREE.LineBasicMaterial( {         
+                color: """,color,""",
+                linewidth: """,edgeThickness,""",
+            } );        
+        """);
+
+        AppendTo(output, "\tconst edges",i," = new Float32Array( [");
+
+        colorPositions := Positions(edgeColors, color);
+        for edge in colorPositions do
+            # if IsEdgeActive(surface, edge, printRecord) then
```

Comment:
Can this commented out line be removed?

Reply:
I would prefer to keep it in, as once schnellecom/SimplicialSurfaces#18 is resolved the check should be reenabled.